build/run-okd.sh

```diff
@@ -4,12 +4,15 @@ OC_USER=${OC_USER:-kubeadmin}
 KUBECONFIG=${1-openshift.local.clusterup/openshift-apiserver/admin.kubeconfig}
 
 echo "Using KUBECONFIG '$KUBECONFIG'"
+echo "Using PROJECT_NAME '${PROJECT_NAME}'"
 
-oc login -u ${USER}
-oc project default
+oc login -u "${OC_USER}"
+oc project "${PROJECT_NAME}"
 oc apply -f deploy/role.yaml
+oc apply -f deploy/clusterrole.yaml
 oc apply -f deploy/service_account.yaml
 oc apply -f deploy/role_binding.yaml
-oc apply -f deploy/operator.yaml
+sed -e "s|namespace:.*|namespace: ${PROJECT_NAME}|" deploy/clusterrole_binding.yaml | oc apply -f -
```

Comment:
This line isn't present on the `master` branch. Do we need it there as well?

Reply:
Yes, this will be part of the #560 and this PR just preparation for it